RELEASE.md

```diff
@@ -36,31 +36,30 @@ All future changes adopted into master will need to be back-ported to those bran
-- The oldest supported release similarly might be `testing-1.15-27` for a prior `stable-1.15-26`.
-- These tag versions, in the form of `major.minor-dist`, only the `major.minor` refer to the kubernetes version. The api within a minor version should not be changing so there should never be a need to refer to the kubernetes patch version.
+- This set of Addons are installed into the SOAK cluster.
 
-**NOTE:** If a breaking change causes a diversion from an older release of Kubernetes to a newer one, prior to tagging the older version must be branched, ie. `stable-1.16-9` would become `stable-1.16`, the changes since the `stable-1.16-9` tag would be merged into this branch, and the new _tag_ would still be `testing-1.16-10` but pointing to the last change on the `stable-1.16` branch.
+### Stable Release (Second and Forth Wednesday)
 
-- This set of Addons are installed into a SOAK cluster<sup>[3](#footnote3)</sup>.
-
-### Stable Release (Weekly, Wednesday)
-
-Each _**Wednesday**_:
+On the second and forth _**Wednesday**_:
 
+- Using automation, parse the PR logs in the `testing` branch for release notes and generate and commit a Changelog.md
+- One _**tag**_ is made for the each supported version of Kubernetes with a consistent semver suffix
+- If the current version of Kubernetes is `1.17.2`, and the last release was `stable-1.17-1.2.x`, the new SOAK tag will be `release-1.17-1.3.0`
+- The same semver portion of the tag `1.3.0` is used for each supported kubernetes version, the new tags being `release-1.16-1.3.0`, and `release-1.15-1.3.0`
+- These tag versions, in the form of `release-<major>.<minor>-<semver>`, only the `<major>.<minor>` refer to the kubernetes version.
+  The api within a minor version should not be changing so there should never be a need to refer to the kubernetes patch version.
 - As a standing agenda item in sig-ksphere-catalog, vote go/no-go on the release of the Addons that have been SOAK tested.
-- Create `stable-` tags for the `testing-` tags that ran in SOAK.
+- Merge the `testing` branch into the `stable` branch<sup>[3](#footnote3)</sup>
 - Announce the release.
 
 <a name="footnote1">1</a>: Based on a two-week soak cycle. If we can have overlapping soak clusters, we can accelerate this.
 
 <a name="footnote2">2</a>: A supported Addon is one which has been tested to work in concert with other Addons in the same release. This suite of Addons, as a whole, constitute a set for which D2iQ customers can get support with their software contract. Variations from the configurations and suite of Addons are not expected to be the responsibility of D2iQ support.
 
-<a name="footnote3">3</a>: At the time of this writing, this process is as yet undetermined as there are no clusters in which to do this SOAK.
+<a name="footnote3">3</a>: In the future, there may need to be multiple `stable` branches as needed to maintain our support commitment.
```

Comment:
This would not only be true for stable branches, given we had the breaking change mentioned above, `master` might be on `2.0.0` but we still need to support `1.x` versions. that said, we might need to have `stable-1.x`, `testing-1.x` _and_ `dev-1.x` branches.